libs/editor/egui_editor/src/apple/ios_ffi.rs

```diff
@@ -418,3 +420,13 @@ pub unsafe extern "C" fn clipboard_paste(obj: *mut c_void) {
     let clip = obj.from_host.clone().unwrap_or_default();
     obj.raw_input.events.push(Event::Paste(clip));
 }
+
+/// # Safety
+/// obj must be a valid pointer to WgpuEditor
+#[no_mangle]
+pub unsafe extern "C" fn indent_at_cursor(obj: *mut c_void, deindent: bool) {
+    let obj = &mut *(obj as *mut WgpuEditor);
+    obj.editor
+        .custom_events
+        .push(Modification::Indent { deindent });
+}
```

Comment:
weird that this is in a different file, but I think we'll be in a better place to organize this when we see what's re-used by an android integration, which might re-use more of the ios_ffi.rs functions

Reply:
This is in a different file since it is not used by macOS.